### PR TITLE
[ready] add a blog of bastiaan

### PR DESCRIPTION
**Pull request by fatboypunk**

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Rebased, updated the share image and the graphic including a big one.

**Comment by RonaldDijkstra** [comment]
Added target blank and updated the social share image (made a typo).

File: source/blog/nl/2016-07-15-samenwerking-op-basis-van-co-creatie-en-duurzaamheid.html.markdown [hunk 1]
```diff
@@ -4,8 +4,8 @@ date: 2016-07-15 11:13 CEST
 tags:
 author: Marian
 lang: nl
-featured: true
-image: images/blog/bevoegd-is-niet-bekwaam.jpg
+featured: false
+image: images/blog/bevoegd-is-niet-bekwaam.png
```
Comment (fatboypunk) [inline source/blog/nl/2016-07-15-samenwerking-op-basis-van-co-creatie-en-duurzaamheid.html.markdown]: please rename this file :)

Comment (RonaldDijkstra) [inline source/blog/nl/2016-07-15-samenwerking-op-basis-van-co-creatie-en-duurzaamheid.html.markdown]: Thnx

**Pull request merged**